receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -195,6 +244,7 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 
 	ghs.rb.SetGitVendorName("github")
 	ghs.rb.SetOrganizationName(ghs.cfg.GitHubOrg)
+	ghs.rb.SetTeamName(ghs.cfg.GitHubTeam)
```

Comment:
What happens if team isn't provided? (it shouldn't be required)

Reply:
It would just have an empty string as its value